portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,4 +120,20 @@ private String getComment(HttpServletRequest request){
       return userComment;
   }
 
+  /** Gets number of comments to print */
+  private int getNumComments(HttpServletRequest request) {
+
+    String numCommentsString = request.getParameter("comment-number");
+    int numComments;
```

Comment:
considering we always assign a value to numComments, we don't need to declare it ahead of time. We can just do int numComments in each case of the if else block. Even better, considering we only make the variable to return it, we can just do returns directly in the if else block.

Reply:
got it!